py/src/braintrust/oai.py

```diff
@@ -263,16 +263,20 @@ def _postprocess_streaming_results(cls, all_results: List[Dict[str, Any]]) -> Di
 
 
 class ResponseWrapper:
-    def __init__(self, create_fn: Optional[Callable[..., Any]], acreate_fn: Optional[Callable[..., Any]]):
+    def __init__(self, create_fn: Optional[Callable[..., Any]], acreate_fn: Optional[Callable[..., Any]], name: str = "openai.responses.create"):
         self.create_fn = create_fn
         self.acreate_fn = acreate_fn
+        self.name = name
 
     def create(self, *args: Any, **kwargs: Any) -> Any:
+        if self.create_fn is None:
+            raise RuntimeError("ResponseWrapper was not properly initialized: 'create_fn' is None. Please ensure you pass a valid create function when constructing ResponseWrapper.")
```

Comment:
we really shouldn't be raising errors in tracing code. can you make this non optional instead?